### ISSUE TITLE
Can't open folders over SFTP unless the server is already in ~/.ssh/known_hosts

### ISSUE THREAD

**Issue by alexhenrie** [lxqt/pcmanfm-qt#1005]
When I type `sftp://kingspeak.chpc.utah.edu/` into the PCManFM-Qt address bar, a dialog appears which states:

Can't verify the identity of "kingspeak.chpc.utah.edu (155.101.26.20)".
This happens when you log in to a computer the first time.

The dialog has two buttons: "Log In Anyway" and "Cancel Login". But it doesn't matter which button I click, either way a new dialog pops up with the error message "Login dialog cancelled". If I try again, the same thing happens. Thus it is impossible to actually access the server from PCManFM.

##### System Information
* Distribution & Version: Arch Linux, current
* Kernel: 5.2.11-arch1-1-ARCH
* Qt Version: 5.13.0
* libfm-qt Version: 0.14.1
* libqtxdg Version: 3.3.1
* lxqt-build-tools Version: 0.6.0
* Package version: 0.14.1-1

**Comment by tsujan** [comment]
Duplicate of https://github.com/lxqt/pcmanfm-qt/issues/108. Quoting from that report: "Once the key was accepted, e. g. by running an SSH session from the shell, SFTP connections work within PCManFM, too."

That may be the reason why nobody has tried to fix it. Most users have the key, so that they have no problem with sftp in pcmanfm-qt (I use it a lot). However, this old bug deserves more attention.

**Comment by agaida** [comment]
We should make a new bug for all problems with remote filesystems - there are several issues and we have a new component to play with -.lxqt-wallet
